packages/framework/ai-collab/src/explicit-strategy/agentEditTypes.ts

```diff
@@ -0,0 +1,175 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { JsonPrimitive } from "./jsonTypes.js";
+
+/**
+ * TODO: The current scheme does not allow manipulation of arrays of primitive values because you cannot refer to them.
```

Comment:
We should probably file follow-up tasks for many of these. I suspect most/all aren't blockers for the alpha release, but they should still be on our radar. Some will certainly be blocked on missing SharedTree functionality - identifying those blockers would be useful.

@taylorsw04 maybe you can help point us to the relevant backlog items that would be blockers for these.

Reply:
AFAICT I don't think any are blocked on new tree functionality. I'd just file them as items in the AI workstream's backlog.

Reply:
Added JSDOC for this in commit b5cceaa3be6163e6a448c08a5b34afbae6af4ee9